content/tutorials/getting-started/index.md

```diff
@@ -15,21 +15,30 @@ lastmod: 2021-08-30T16:00:00+02:00
 
 In this tutorial, we'll describe every step to help you run your first test with Gatling Enterprise.
 
-{{< alert tip >}}
+{{< alert info >}}
 **Requirements**
 
 * A valid [GitHub](https://github.com/) account
 * A Java Development Kit (JDK) installed in version 8 or 11
-* The [Gatling bundle](https://gatling.io/open-source/#downloadgatling) as a starter or an existing Maven/Gradle/SBT Gatling project
+{{< /alert >}}
+
+{{< alert tip >}}
+**Recommendations**
+
+* Using the [Gatling bundle](https://gatling.io/open-source/#downloadgatling) is beginner friendly.
+* For long term projects, consider using a Gradle, Maven or SBT Gatling project. You can download some Gatling Enterprise ready sample projects from the Cloud dashboard directly!
 {{< /alert >}}
 
 Gatling is a code based load testing tool. It is based on the Scala programming language and is an Open-Source project that is free to use! Gatling is used to power Gatling Enterprise in order to make it a full-fledged load testing tool with even more features, such as live reports, advanced metrics and large scale clustered tests. The tests you code with Gatling are compatible with Gatling Enterprise without any changes.
 
 Gatling requires a Java installation so make sure that you have a JDK 8 or 11 installed on your computer. [Adoptium](https://adoptium.net) provides good packages that contain everything you need.
 
-While there are a lot of ways to start working on your first test, **if you are new to this, we recommend using the Gatling bundle**. It is self sufficient and doesn't require knowledge in any of the usual Java/Scala build tools, i.e. Gradle, Maven or SBT.
+While there are a lot of ways to start working on your first test, **if you are new to this or not a developer, we recommend using the Gatling bundle as it is beginner friendly**. It is self sufficient and doesn't require knowledge in any of the usual Java/Scala build tools, i.e. Gradle, Maven or SBT.
```

Comment:
While there are a lot of ways to create your first test, **if you are new to load testing or not a developer, we recommend using the Gatling bundle as it is beginner friendly**. It is self sufficient and doesn't require knowledge of the usual Java/Scala build tools, i.e. Gradle, Maven or SBT.